core/geometry/src/test/serialization/BSIJSON.test.ts

```diff
@@ -66,7 +66,7 @@ function isDifferentTypeName(obj: object, noisy: boolean = false): boolean {
   if (obj.constructor) {
     if (obj.constructor !== previousConstructor) {
       if (noisy)
-        console.log("First ", name);
+        console.log("First ", obj.constructor.name);

```

Comment:
@EarlinLutz This probably should have been an error before, but it was actually using a global `name` var from lib.dom.d.ts before.  That name var has now been deprecated, so I had to guess at what this was supposed to be logging...

Reply:
that's fine